templates/gallery/gallery-inner.amp.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gallery": {
+    "head-tag": {
+      "title": "Gallery Template",
+      "canonical-path": "templates/gallery-inner.amp",
+      "css-path": "templates/gallery/page.css",
+      "font-stylesheets": [
+        "https://fonts.googleapis.com/css?family=Poppins:400,500",
+        "https://fonts.googleapis.com/css?family=Oswald:400"
+      ]
+    },
+    "hero": {
+      "heading": "Living Tree",
+      "subheading": "<time datetime=\"2017-08-24\">Sept 24</time>-<time datetime=\"2017-08-29\">29</time> / Anna Moss",
```

Comment:
i would prefer component reuse over rewrite. I just took a look at the component i think it is very much reusable -